example/main.go

```diff
@@ -46,10 +46,13 @@ func main() {
 	runtime.SetMutexProfileFraction(5)
 	runtime.SetBlockProfileRate(5)
 	pyroscope.Start(pyroscope.Config{
-		ApplicationName: "simple.golang.app-new",
-		ServerAddress:   "http://localhost:4040", // this will run inside docker-compose, hence `pyroscope` for hostname
-		Logger:          pyroscope.StandardLogger,
-		AuthToken:       os.Getenv("PYROSCOPE_AUTH_TOKEN"),
+		ApplicationName:   "simple.golang.app-new",
+		ServerAddress:     "http://gouser:gopass@localhost:4040/",
```

Comment:
Isn't this url confusing since basic auth should be set via `BasicAuth{User,Password}`?

Reply:
yeah, it is a leftover. At first we did not plan to add separate configuration for basic auth since url configuration is possible but latter it was added. I will remove it